src/main/java/net/snowflake/ingest/example/SnowflakeIngestBasicExample.java

```diff
@@ -36,10 +36,10 @@
   // pre-requesites
   private static String account = "s3testaccount";
```

Comment:
All values in this file should effectively remain unchanged. This is what customers will see in their examples, so we should not be exposing internal testing elements.